### PR TITLE
docs(release-notes): Add retroactive mention of User Info Fetcher being experimental

### PR DESCRIPTION
**Pull request by NickLarsenNZ**
Add a note to the release notes about UIF being experimental (since it was announced in 24.3)

### CODE REVIEW

File: modules/ROOT/pages/release-notes.adoc [hunk 1]
```diff
@@ -103,6 +103,8 @@ Documentation::
 
 * Apache Spark: documentation has been added showing how to provision Spark dependencies
 
+* Open Policy Agent: N.B. As mentioned in the release 24.3, we will be actively building out the backends supported by the User Info Fetcher. This feature should be therefore be treated as experimental, as implementation details are subject to change as we consolidate and fine-tune the tool.
```
Comment (nightkr) [inline modules/ROOT/pages/release-notes.adoc:106]: "Implementation details" is a bit too narrow. We're always free to change implementation details, the problem is that there is no public API yet...

Comment (adwk67) [reply modules/ROOT/pages/release-notes.adoc:106]: how about "usage details" or is that too clunky?

Comment (nightkr) [reply modules/ROOT/pages/release-notes.adoc:106]: Feels clunky but it could work.

Comment (adwk67) [reply modules/ROOT/pages/release-notes.adoc:106]: Or just:

> This feature should be therefore be treated as experimental as we continue to extend and consolidate back-end handling and fine-tune the tool in general.

Comment (nightkr) [reply modules/ROOT/pages/release-notes.adoc:106]: :+1:

Comment (adwk67) [reply modules/ROOT/pages/release-notes.adoc:106]:
```suggestion
* Open Policy Agent: N.B. As mentioned in the release 24.3, we will be actively building out the backends supported by the User Info Fetcher.
This feature should be therefore be treated as experimental as we continue to extend and consolidate back-end handling and fine-tune the tool in general.
```

**Review by adwk67 — APPROVED** [review]

**Review by adwk67 — APPROVED** [review]

**Pull request merged**